### DISCUSSION TITLE
🎅Sorteio de Natal FrontendBR - 3 cursos ninjas do Daciuk 💙 (versão 2020)

### DISCUSSION THREAD

**Discussion by fdaciuk** [frontendbr/forum#1931] [Dicas]
Algumas pessoas estavam comentando [nesse post](https://github.com/frontendbr/forum/discussions/1310) de 2 anos atrás, onde nós fizemos o sorteio de 3 [cursos Ninja](https://www.udemy.com/user/fernando-daciuk/). Como eu percebi que tem gente interessada nos cursos esse ano, vamos fazer de novo, na versão 2020 haha \o/

As regras são as mesmas:

Basta responder essa issue escolhendo entre:

- JavaScript Ninja
- React Ninja
- Git e Github

No dia **20/12/2020** (domingo), vamos sortear as três pessoas ganhadoras.

(o sorteio não será por curso, e sim por pessoa, que vai ganhar o curso escolhido)

Sensacional né?

Está valendo e feliz natal pra todos nós! ❤️ 

### EDIT: O sorteio será hoje (20/12) às 17h (horário de Brasília) nos meus canais da Twitch e YouTube:

Twitch: https://twitch.tv/fdaciuk
YouTube: https://youtube.com/queroserninja

### EDIT (2): 

# Ganhadores:
E o sorteio foi feito! Pra quem não acompanhou, o link da transmissão foi esse:
https://www.youtube.com/watch?v=qr75i43DGNw

tada Parabéns aos ganhadores: tada

@CauaHenrique19 - Git e GitHub Ninja
@CarolineSarre - React Ninja
@im-diego - JavaScript Ninja

**Comment by gusleaooliveira** [comment]
React Ninja

**Reply by alepeglow** [reply]
Javascript Ninja

**Reply by GabrielaSchneider1** [reply]
JavaScript Ninja

**Reply by fdaciuk** [reply]
@alepeglow e @GabrielaSchneider1 criem um comentário novo, pois respostas não vão contar :)

**Reply by KeziAntero** [reply]
JavaScript Ninja

**Reply by emersonpessoa01** [reply]
JavaScript Ninja

**Comment by rodrigoalmeidagit** [comment]
React Ninja

**Comment by dnstld** [comment]
React Ninja

**Comment by pbruny** [comment]
React Ninja

**Comment by w3b3** [comment]
React Ninja

**Comment by caiohbfurtado** [comment]
React Ninja

**Comment by GoharaJunior** [comment]
React Ninja

**Comment by thejpdev** [comment]
React ninja

**Comment by JoaoPauloLisboa** [comment]
React Ninja

**Comment by luansemensato** [comment]
React Ninja

**Comment by AnthonyTailer** [comment]
React Ninja

**Comment by joseemds** [comment]
React ninja

**Comment by FlexaBR** [comment]
React Ninja

**Comment by abraaoduarte** [comment]
React Ninja

**Comment by ghost** [comment]
React Ninja

**Comment by GeisonRDS** [comment]
React Ninja

**Comment by CaioAugustoo** [comment]
React Ninja

**Comment by cristianefariadev** [comment]
JavaScript Ninja

**Comment by lciceris** [comment]
React Ninja

**Comment by biliesilva** [comment]
React Ninja

**Comment by Pedro-phd** [comment]
JavaScript Ninja :D

**Comment by iaurg** [comment]
React Ninja

**Comment by SttavoS** [comment]
React Ninja

**Comment by rebecagrn** [comment]
React Ninja

**Comment by diegosilvadigital** [comment]
React Ninja

**Comment by dan-liberato** [comment]
React Ninja

**Comment by brunoromeiro** [comment]
React Ninja

**Comment by wagnnercsl** [comment]
JavaScript Ninja

**Comment by igorconde** [comment]
React Ninja

**Comment by igorgms** [comment]
JavaScript Ninja

**Comment by tiagocalisto** [comment]
Java Script Ninja

**Comment by painttw07** [comment]
JavaScript Ninja

**Comment by rafoccb** [comment]
React Ninja

**Comment by jjunior96** [comment]
React Ninja

**Comment by pcampina** [comment]
React Ninja

**Comment by damoclesgil** [comment]
React Ninja

**Comment by luizeboli** [comment]
JavaScript Ninja

**Comment by RGalli** [comment]
JavaScript Ninja

**Comment by ranzolinrafa** [comment]
React Ninja

**Comment by wilsonneto-swe** [comment]
React Ninja

**Comment by StefanyVasc** [comment]
React Ninja

**Comment by hspotted** [comment]
React Ninja

**Comment by thallestorma** [comment]
JavaScript Ninja

**Comment by nicolas-justin** [comment]
React Ninja

**Comment by alicefrancener** [comment]
React Ninja

**Comment by Morpa** [comment]
React Ninja

**Comment by larismourullo** [comment]
React Ninja

**Comment by hevertonsb** [comment]
React Ninja

**Comment by davidalves1** [comment]
JavaScript Ninja

**Comment by karluslima** [comment]
React Ninja

**Comment by Paulo-Martin182** [comment]
React Ninja

**Comment by jennysousa** [comment]
React Ninja

**Comment by MarciaA0693** [comment]
React Ninja

**Comment by FernandoFreireDev** [comment]
React Ninja

**Comment by bhayron** [comment]
React Ninja

**Comment by LucasMallmann** [comment]
React Ninja

**Comment by valdemirfilho** [comment]
Javascript Ninja

**Comment by gdiasb** [comment]
Javascript Ninja

**Comment by BrunoLagoa** [comment]
Aprenda a criar sites animados com Greensock

**Reply by fdaciuk** [reply]
@BrunoLagoa esse curso é do Justen! Posta na [issue dele](https://github.com/frontendbr/forum/discussions/1932) :)

**Comment by LeonardoMarco** [comment]
React Ninja

**Comment by felrespin** [comment]
React Ninja

**Comment by staelsabrina** [comment]
React Ninja

**Comment by cdfortes** [comment]
React Ninja

**Comment by gustavo-brandao** [comment]
JavaScript Ninja

**Reply by asdiasx** [reply]
javascript ninja

**Reply by fdaciuk** [reply]
@asdiasx cria um comentário separado, senão pode não contabilizar a sua participação :D

**Comment by jonatasbr** [comment]
React Ninja

**Comment by AndreGM** [comment]
React Ninja

**Comment by waldenermonteiro** [comment]
React Ninja

**Comment by danielacb** [comment]
React Ninja

**Comment by pamelakelly0023** [comment]
React ninja

**Comment by jeanmendescs** [comment]
Git e Github

**Comment by LucieneSousa** [comment]
JavaScript Ninja

**Comment by adailsonaguiar** [comment]
Javascript ninja

**Comment by gabsdotco** [comment]
React Ninja

**Comment by Brunofsi** [comment]
React Ninja

**Comment by marqueslu** [comment]
React Ninja

**Comment by Henrique018** [comment]
JavaScript Ninja

**Comment by jeffersonbraster** [comment]
React Ninja

**Reply by themisterpaps** [reply]
Javascript ninja

**Comment by rafaabatistas** [comment]
JavaScript Ninja

**Comment by diegodario88** [comment]
React Ninja

**Comment by sleduardo20** [comment]
React Ninja

**Comment by diiegoburiti** [comment]
JavaScript Ninja

**Comment by felipemv13** [comment]
React Ninja

**Comment by rodrigodasilva** [comment]
React ninja

**Comment by christophermoura** [comment]
React Ninja

**Comment by ianfelix** [comment]
React Ninja

**Comment by andresavasconcelos** [comment]
JavaScript Ninja

**Comment by viniciusmattosrj** [comment]
JavaScript Ninja

**Comment by kaiofelipejs** [comment]
React Ninja

**Comment by malnn** [comment]
React Ninja

**Comment by fukudathais** [comment]
React Ninja

**Comment by kauanmocelin** [comment]
React Ninja

**Comment by fredmosc** [comment]
React Ninja

**Comment by RomuloBraga92** [comment]
React Ninja

**Comment by OoNascimentoOo** [comment]
React Ninja

**Comment by samuelvicentini** [comment]
React Ninja

**Comment by Roneycacau** [comment]
React Ninja

**Comment by eltonbrayner** [comment]
React Ninja

**Comment by ppfurtado** [comment]
Git e Github

**Comment by mutheusalmeida** [comment]
React Ninja

**Comment by akemimeka** [comment]
React Ninja!

**Comment by daniellucindo** [comment]
React Ninja

**Comment by allanco91** [comment]
JavaScript Ninja

**Comment by marceloedu2** [comment]
React Ninja

**Comment by jrbytes** [comment]
React Ninja

**Comment by marcuspaulo** [comment]
React Ninja

**Comment by nelsonmfinda** [comment]
React Ninja

**Comment by juligaioso** [comment]
react ninja :)

**Comment by carmo-sousa** [comment]
React Ninja

**Comment by pikonha** [comment]
react ninja

**Comment by capelaum** [comment]
React Ninja

**Comment by jonataspinto** [comment]
React Ninja

**Comment by FredSoares** [comment]
React Ninja 😉

**Comment by Renato66** [comment]
React ninja

**Comment by GuilleAngulo** [comment]
React Ninja

**Comment by vagnerkuntz** [comment]
React Ninja

**Comment by rberts** [comment]
React Ninja

**Comment by palhanos** [comment]
React Ninja

**Comment by paulohfreire** [comment]
JavaScript Ninja

**Comment by IggorSantos** [comment]
JavaScript Ninja

**Comment by aintluks** [comment]
React Ninja

**Comment by Xandowski** [comment]
React Ninja

**Comment by mauriciomutte** [comment]
Git e Github

**Comment by leonardobrito** [comment]
JavaScript Ninja

**Comment by RuanAyram** [comment]
JavaScript Ninja

**Comment by pricillaragao-zz** [comment]
React Ninja

**Comment by plcosta** [comment]
JavaScript Ninja

**Comment by leonardotdleal** [comment]
React Ninja

**Comment by Brenont** [comment]
JavaScript Ninja

**Comment by pietra-nunes** [comment]
React Ninja

**Comment by Bsignx** [comment]
React Ninja

**Comment by magnoliamedeiros** [comment]
React Ninja

**Comment by LuizEduardoBP** [comment]
React Ninja

**Comment by joaopfc** [comment]
JavaScript Ninja

**Comment by CibaYumi** [comment]
JavaScript Ninja

**Comment by alabliuk** [comment]
JavaScript Ninja

**Comment by igortullio** [comment]
React Ninja

**Comment by caiohenrique-developer** [comment]
React Ninja

**Comment by sousaf4** [comment]
React Ninja

**Comment by deyvisonborges** [comment]
React Ninja

**Comment by walefe** [comment]
JavaScript Ninja.

**Comment by brunoamil** [comment]
React Ninja

**Comment by beatriz1304** [comment]
JavaScript Ninja

**Comment by williamjamir** [comment]
JavaScript Ninja

**Comment by Steven-J-Alves** [comment]
JavaScript Ninja

**Comment by helderberto** [comment]
Git e Github

**Comment by luucasfarias** [comment]
JavaScript Ninja

**Comment by paulequilibrio** [comment]
JavaScript Ninja

**Comment by giovannism20** [comment]
React Ninja

**Comment by luysla** [comment]
JavaScript Ninja

**Comment by valdelirio** [comment]
Git e GitHub

**Comment by isabelffernandes** [comment]
> * Git e Github

Git e Github

**Comment by rickletras** [comment]
JavaScript Ninja

**Comment by kivervinicius** [comment]
React Ninja

**Comment by Lucaslpa** [comment]
React Ninja

**Comment by jaimebs** [comment]
React Ninja

**Comment by Emerson-developer** [comment]
React Ninja.

**Comment by GabrielGuedess** [comment]
Git e Github

**Comment by lucas-barbosa** [comment]
React Ninja

**Comment by tiagooliveira08** [comment]
React ninja

**Comment by Deisemaya** [comment]
React Ninja, bem no dia do meu niver, hehe

**Comment by rafaelfachinelli** [comment]
React Ninja

**Comment by raphabarreto** [comment]
JavaScript Ninja

**Reply by EmanuelleViana** [reply]
React Ninja

**Comment by jessicasilvano** [comment]
React Ninja

**Comment by luizbalisa** [comment]
React Ninja

**Comment by EdsonLuiz** [comment]
JavaScript Ninja

**Comment by Rychillie** [comment]
JavaScript Ninja

**Comment by ghost** [comment]
React Ninja

**Comment by BLRocha** [comment]
JavaScript Ninja

**Comment by StefanoSaffran** [comment]
React Ninja

**Comment by jhenrique-dev** [comment]
React Ninja

**Comment by timabuntu** [comment]
git e github

**Comment by reubber** [comment]
Javascript Ninja

**Comment by diegobarretoo** [comment]
Javascript Ninja

**Comment by hgtpcastro** [comment]
Git e Github

**Comment by felipprodrigues** [comment]
React Ninja

**Comment by lucianofa** [comment]
React Ninja

**Comment by dauryellen** [comment]
React Ninja

**Comment by imarinho-dev** [comment]
React Ninja

**Comment by ArmandoVinicius** [comment]
React Ninja

**Comment by renatoknot** [comment]
React Ninja

**Comment by cassiodourado1** [comment]
JavaScript Ninja

**Comment by FellipySantos** [comment]
Git e Github

**Comment by JorgeSegtowich** [comment]
Git e Github

**Comment by devfreitag** [comment]
React Ninja 🐱‍👤

**Comment by CauaHenrique19** [comment]
Git e Github

**Comment by jacksonks** [comment]
React Ninja

**Comment by w3nder** [comment]
React Ninja

**Comment by saulobraine** [comment]
React Ninja

**Comment by CezarGarrido** [comment]
Git e Github

**Comment by daviwesley** [comment]
React Ninja

**Comment by marciot2** [comment]
React Ninja

**Comment by alexandremaeda** [comment]
JavaScript Ninja

**Comment by victor1162** [comment]
React Ninja

**Comment by caroline-rodrigues** [comment]
React Ninja

**Comment by netojocelino** [comment]
React Ninja

**Comment by wellpinho** [comment]
React Ninja! \o/ \o/

**Comment by ThiagoKachi** [comment]
JavaScript Ninja

**Comment by alepeglow** [comment]
Javascript Ninja

**Comment by GabrielaSchneider1** [comment]
JavaScript Ninja

**Comment by victor1162** [comment]
React ninja

**Comment by gustavocastrow** [comment]
React Ninja

**Comment by antoniomalves** [comment]
React Ninja

**Comment by CarolineSarre** [comment]
React Ninja

**Comment by andersencypriano** [comment]
React Ninja =]

**Comment by dgldaniel** [comment]
JavaScript Ninja

**Comment by crishenrique** [comment]
Git e Github

**Comment by albertomoiseis** [comment]
Javascript Ninja

**Comment by wilsonfaustino** [comment]
JavaScript Ninja

**Comment by PatrickZava** [comment]
React Ninja

**Comment by hedenica** [comment]
Javascript Ninja

**Comment by fernandorssa** [comment]
React Ninja

**Comment by rafaumeu** [comment]
[ReactJs Ninjs](https://www.udemy.com/curso-reactjs-ninja/?couponCode=BF2020)

**Comment by dav1rm** [comment]
React Ninja

**Comment by rodrigophpweb** [comment]
JavaScript Ninja

**Comment by lucianovieiravilella** [comment]
JavaScript Ninja

**Comment by brabao** [comment]
JavaScript Ninja

**Comment by rijema** [comment]
React Ninja

**Comment by jpdrsanchez** [comment]
React Ninja

**Comment by milenayamamoto** [comment]
React Ninja!

**Comment by helcioogomes** [comment]
React Ninja

**Comment by jonathanwdev** [comment]
react ninja

**Comment by marcelosanto** [comment]
JavaScript Ninja

**Comment by Grazinascito** [comment]
javascript ninja

**Comment by felrespin** [comment]
React Ninja

**Comment by mariana-caetano** [comment]
JavaScript Ninja

**Comment by priscillasantana** [comment]
Javascript ninja

**Comment by brunoardel** [comment]
JavaScript Ninja

**Comment by yucianci** [comment]
Git e Github

**Comment by lincolncarvalho** [comment]
Git e Github

**Comment by galvaodev** [comment]
React Ninja

**Comment by CesarBalzer** [comment]
JavaScript Ninja

**Comment by Lucaalmeida** [comment]
JavaScript Ninja

**Comment by Lucaalmeida** [comment]
JavaScript Ninja

**Comment by arsantin** [comment]
React Ninja

**Comment by Everton10info** [comment]
React Ninja

**Comment by Ana-MP** [comment]
JavaScript Ninja

**Comment by thihuli** [comment]
Javascript ninja

**Comment by ZankoyDll1999** [comment]
React ninja

**Comment by michelspirlandeli** [comment]
React ninja

**Comment by gabriel-portuga** [comment]
React Ninja

**Comment by HeloiseSantos** [comment]
JavaScript Ninja

**Comment by EmersonGomes21** [comment]
React Ninja

**Comment by efeeseao** [comment]
React Ninja

**Comment by alexsanderfer** [comment]
React Ninja

**Comment by robsonnatanael** [comment]
Javascript Ninja

**Reply by rosampe** [reply]
React Ninja

**Reply by fdaciuk** [reply]
Oi @rosampe! Faz um comentário separado, por favor? Repostas não vão contar no sorteio :)

**Comment by italogois** [comment]
JavaScript Ninja

**Comment by williamfelippe** [comment]
JavaScript Ninja

**Comment by ghost** [comment]
Javascript ninja. :D

**Comment by gomesgr** [comment]
Git e Github

**Comment by GustavoBrtos** [comment]
React Ninja

**Comment by AndreVeiga** [comment]
React Ninja

**Comment by arnald7** [comment]
React Ninja

**Comment by andreiv-araujo** [comment]
Javascript ninja

**Comment by fdaciuk** [comment]
E o sorteio foi feito! Pra quem não acompanhou, o link da transmissão foi esse:
https://www.youtube.com/watch?v=qr75i43DGNw

🎉  Parabéns aos ganhadores: 🎉 

@CauaHenrique19 - Git e GitHub Ninja
@CarolineSarre - React Ninja
@im-diego - JavaScript Ninja

Entrem em contato comigo via DM no [meu Twitter](https://twitter.com/fdaciuk) pra gente combinar a entrega dos cupons ❤️
